workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -347,14 +362,20 @@ to maintain high networking performance.
 > SR-IOV network, you have to also request corresponding devices from the
 > device plugin, by adding appropriate `resources.limits` and
 > `resources.requests` entries.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
       interfaces:
         - name: sriov-net
           sriov: {}
-   resources:
-     limits:
-       intel.com/sriov: "1"
-     requests:
-       intel.com/sriov: "1"
+    resources:
```

Comment:
Hm, I fear that there is one whitespace more required. Can that be?

Reply:
@rmohr I've checked locally, it seems to consistently use 2-space indent for all levels.

Reply:
I see. My mistake. Good to go then. :+1: